_docs/_latest/setup/manual-deployment.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Manual Deployment
+order: 004
+---
+
+# {{ page.title }}
+
+This page describes the steps to install and deploy Forseti manually on 
+Google Cloud Platform (GCP). It's best to use the
+[automated installer]({% link _docs/latest/setup/install.md %})
+for an easy and error-free deployment. Use this guide only if you strongly
+prefer to install Forseti yourself, or if you want to learn about the
+installation process.
+
+To complete this guide, you will need in-depth knowledge of GCP. For example,
+you will need to know how to use the gcloud command-line tool and where to bind
+a service account to a VM in the GCP Console. This guide refers to the installer
+and deployment templates for specific details of the commands to use.
+
+---
+
+## Create a project
+
+Create a new project to host Forseti. Forseti is intended to run in its own
+dedicated project so that access to its highly privileged permissions can be
+controlled.  Assign a billing account to the project.
+
+### Enable APIs
+
+For information about the APIs you need to enable in the project that hosts
+Forseti, see [required APIs]({% link _docs/latest/required_apis.md %}).
+
+## Deploy a server VM
+
+### Create a Forseti server service sccount
+
+Create a server service account by running the following command:
+
+```
+gcloud iam service-accounts create forseti-server-gcp-####### --display-name forseti-server-gcp-#######
+```
+
+Where `#######` is a random alphanumeric unique identifier.
+
+### Assign roles
+
+For information about the roles you need to assign to the Forseti server
+service account, see 
+[the server service account]({% link _docs/latest/concepts/service-accounts.html#the-server-service-account %}).
+
+### Create a Forseti server VM instance
+
+* For information about the specifications needed for a Forseti server
+  VM instance, see the [Compute Engine deployment template](https://github.com/GoogleCloudPlatform/forseti-security/blob/stable/deployment-templates/compute-engine/server/forseti-instance-server.py)
+* When you create the instance, set the service account of the instance to the service account we just created or 
+if you already created the instance, you can [update the service account](https://cloud.google.com/compute/docs/access/create-enable-service-accounts-for-instances#changeserviceaccountandscopes).
+
+### Install the Forseti Server
+
+To install the Forseti Server, follow the steps below:
+
+1. SSH into the server VM
```

Comment:
How do I do this? Are we connecting to the instance (https://cloud.google.com/compute/docs/instances/connecting-to-instance) or something else?

Reply:
Updated with command.

Reply:
Thanks!